packages/node/test/integration/plugins/operator/OperatorPlugin.test.ts

```diff
@@ -33,7 +32,7 @@ describe('OperatorPlugin', () => {
     beforeAll(async () => {
         const deployment = (await setupOperatorContract({
             nodeCount: 1,
-            createTestWallet
+            deployOperatorContract: _operatorContractUtils.deployOperatorContract
```

Comment:
this indirection isn't really explained anywhere... why is deployOperatorContract an argument? It's passed in the same way everywhere, so it might as well not be passed at all.

Reply:
That function is from `sdk` and test-utils can't have `sdk` as a dependency (as it has the dependency other way around: `test-utils` is a dependency of `sdk`)

It was explained in the original PR which added that indirection: https://github.com/streamr-dev/network/pull/2963